src/main/scala/overflowdb/codegen/Helpers.scala

```diff
@@ -24,6 +24,7 @@ object Helpers {
       case Cardinality.One       => HigherValueType.None
       case Cardinality.ZeroOrOne => HigherValueType.Option
       case Cardinality.List      => HigherValueType.List
+      case _=> ???
```

Comment:
shouldn't this be `ISeq` as well?

Reply:
The HigherValueType representation is only used for edge properties -- and we don't need to support array-valued edge properties.

This is not just lazyness: Either we use and test `array`/`iseq` cardinality for at least one edge property, or the code will rot away.